tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -176,9 +190,9 @@ namespace EnergyPlus {
 			"  OutdoorAir:Mixer,         !- Outside Air Mixer Object Type",
 			"  TU1 OA Mixer,             !- Outside Air Mixer Object Name",
 			"  COIL:Cooling:DX:VariableRefrigerantFlow,  !- Cooling Coil Object Type",
-			"  TU1 VRF DX Cooling Coil,  !- Cooling Coil Object Name",
+			"  TU1 VRF DX Cool Coil,  !- Cooling Coil Object Name",
```

Comment:
Simply changed cool coil name to avoid crash of duplicate names, clear_state issue.